### DISCUSSION TITLE
Was logging completely removed in v5?

### DISCUSSION THREAD

**Discussion by luis-fss** [litedb-org/LiteDB#2070] [Q&A]
I would like to make some logging, looked in the documentation and didn't find anything, then I found this issue: https://github.com/mbdavid/LiteDB/issues/776

But this api is not available in v5, so has logging been completely removed since v4?

New reference issue: https://github.com/mbdavid/LiteDB/issues/2086

**Comment by guillaume86** [comment]
I'm interested in this, more precisely I'd like to hook LiteDB to MiniProfiler and I think a good logging API would be the right way to do it.
